dracut/30ignition/module-setup.sh

```diff
@@ -57,6 +69,17 @@ install() {
     inst_simple "$moddir/ignition" \
         "/usr/bin/ignition"
 
+    # Rule to allow udev to discover unformatted encrypted devices
+    inst_simple "$moddir/99-xx-coreos-systemd-cryptsetup.rules" \
+        "/usr/lib/udev/rules.d/99-xx-coreos-systemd-cryptsetup.rules"
+
+    # Clevis requires cracklib_dicts for entropy, copy only a gzipped copy
+    # of cracklib-small.pwd & cracklib-small.pwi
```

Comment:
We could custom-generate an empty dictionary to save 100 KB.  But I think it should be sufficient to add `/etc/security/pwquality.conf.d/ignition-luks.conf` to the initramfs containing e.g.:

```
# We don't ship cracklib dicts in the initramfs, so don't check
# generated clevis keys against them
dictcheck = 0
```